commonknowledge/mapped
# Feature/choose data source to update choropleth

## Description
- Adds area stats on the UI
- Automatic "rolling up" of geographic area stats into larger geographic areas
- Some map style tweaks

## How to test
- Checkout branch on your machine, run `poetry` and `npm` commands to install any missing dependencies
- Launch the app, go to `/data-sources`, select google sheets and import from this sheet ID: **1XA5gvZNOwAqe_YPJw6uCeNB0SbTRYDwNceaWOOq8rvE** & sheet name **UK Wards May 2024 - FAKE VOTES**
- Select data type **area stats**, type of location **ward**, ward field **WD24CD**, unique field **ObjectId** title field **WD24NM**
- Import the data
- Go to `/reports` and create a new report
- Add data source -> select your imported data -> add data source
- Go to the configuration tab
- Select the data field to visualise the data
- Change the political boundary to see the data aggregation between boundary types
